tests/byzantium/eip196_ec_add_mul/test_ecadd.py

```diff
@@ -27,8 +27,106 @@
             Spec.G1,
             id="generator_plus_inf",
         ),
+        pytest.param(
+            Spec.INF_G1 + Spec.G1,
+            Spec.G1,
+            id="inf_plus_generator",
+        ),
+        pytest.param(
+            Spec.G1 + Spec.INF_G1,
```

Comment:
```suggestion
            Spec.G1 + Spec.INF_G1 + Spec.INF_G1,
```

From `ecadd_1-2_0-0_21000_192Filler.json`